### PR TITLE
Update `ko` localized messages

### PR DESCRIPTION
**Pull request by Gudahtt**
These translations were provided by Lionbridge.

### CODE REVIEW

**Comment by Gudahtt** [comment]
If any community contributors would like to help review this PR, it would be greatly appreciated!

I'll tag anyone here that has contributed in the past: @x86kernel @bowbowbow @nujabes403 @hackmod @neutiyoo 

I wanted to give you all the opportunity to provide feedback because we owe our current translations to you. Please don't feel obliged if you don't have the time to review, and please let me know if you'd like me to stop tagging you for reviews in the future.

Thanks!

File: app/_locales/ko/messages.json [hunk 1]
```diff
@@ -503,84 +803,128 @@
...
   "localhost": {
-    "message": "로컬호스트 8545"
+    "message": "Localhost 8545"
   },
   "lock": {
-    "message": "로그아웃"
+    "message": "잠금"
+  },
+  "lockTimeTooGreat": {
+    "message": "자금 시간이 너무 김"
```
Comment (neutiyoo) [inline app/_locales/ko/messages.json:927]: '자금' -> '잠금'

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by nujabes403** [comment]
LGTM

File: app/_locales/ko/messages.json (hunk 1 above)
Comment (Gudahtt) [reply app/_locales/ko/messages.json:927]: Thanks, updated!
```suggestion
    "message": "잠금 시간이 너무 김"
```

File: app/_locales/ko/messages.json [hunk 2]
```diff
@@ -592,206 +936,318 @@
...
+    "description": "The $1 is the bolded word 'Never', from 'metametricsCommitmentsBoldNever'"
+  },
+  "metametricsCommitmentsNeverCollectKeysEtc": {
+    "message": "$1은(는) 키, 주소, 거래, 잔액, 해시 또는 개인 정보를 수집합니다.",
+    "description": "The $1 is the bolded word 'Never', from 'metametricsCommitmentsBoldNever'"
+  },
+  "metametricsCommitmentsNeverSellDataForProfit": {
+    "message": "$1은(는) 수익을 위해 데이터를 판매합니다. 절대로요!",
+    "description": "The $1 is the bolded word 'Never', from 'metametricsCommitmentsBoldNever'"
+  },
```
Comment (neutiyoo) [inline app/_locales/ko/messages.json:983]: We are replacing `$1` with `Never` which is the value from `metametricsCommitmentsBoldNever`.
For example, `"$1 collect your full IP address"` will be `"Never collect your full IP address"`. This makes sense.

However, in Korean, we are replacing `$1` with `안 함` which is the value from `metametricsCommitmentsBoldNever`.

a) `$1은(는) 전체 IP 주소를 수집하지 않습니다.`
b) `안 함은(는) 전체 IP 주소를 수집하지 않습니다.`

and the a) will be the b) after `$1` is replaced with `안 함`.

In the case, the sentence won't make sense as I see it.

A possible solution can be the following:

we set `$1` as `하지 않습니다` and change the related ones as the following

- `전체 IP 주소를 수집$1.` (`전체 IP 주소를 수집하지 않습니다.`)
- `키, 주소, 거래, 잔액, 해시 또는 개인 정보를 수집$1.` (`키, 주소, 거래, 잔액, 해시 또는 개인 정보를 수집하지 않습니다.`)
- `수익을 위해 데이터를 판매$1.` (`수익을 위해 데이터를 판매하지 않습니다.`)

**Review by neutiyoo — CHANGES_REQUESTED** [review]

File: app/_locales/ko/messages.json (hunk 2 above)
Comment (neutiyoo) [inline app/_locales/ko/messages.json:983]: ## Background

We are replacing `$1` with `Never` which is the value from `metametricsCommitmentsBoldNever`.
For example, `"$1 collect your full IP address"` will be `"Never collect your full IP address"`. This makes sense.

## Problem
However, in Korean localization, we are replacing `$1` with `안 함` which is the value from `metametricsCommitmentsBoldNever`.

a) `$1은(는) 전체 IP 주소를 수집하지 않습니다.`
b) `안 함은(는) 전체 IP 주소를 수집하지 않습니다.`

The a) will be the b) after `$1` is replaced with `안 함`.
In the case, the sentence won't make sense as I see it.

## Solution
Therefore, the following should be changed:
- `metametricsCommitmentsBoldNever`
- `metametricsCommitmentsNeverCollectIP`
- `metametricsCommitmentsNeverCollectKeysEtc`
- `metametricsCommitmentsNeverSellDataForProfit`

A possible solution can be the following:

we can set `$1` as `하지 않습니다` and change the related ones as the following:

- `전체 IP 주소를 수집$1.` (`전체 IP 주소를 수집하지 않습니다.`)
- `키, 주소, 거래, 잔액, 해시 또는 개인 정보를 수집$1.` (`키, 주소, 거래, 잔액, 해시 또는 개인 정보를 수집하지 않습니다.`)
- `수익을 위해 데이터를 판매$1.` (`수익을 위해 데이터를 판매하지 않습니다.`)

**Review by ghost — CHANGES_REQUESTED** [review]

File: app/_locales/ko/messages.json (hunk 2 above)
Comment (Gudahtt) [reply app/_locales/ko/messages.json:983]: Great catch!

Either one of those options seem fine to me :thinking: Which do you think would be more idiomatic/commonplace?

Comment (neutiyoo) [reply app/_locales/ko/messages.json:983]: The second one looks good to me.

**Review by ghost — CHANGES_REQUESTED** [review]

File: app/_locales/ko/messages.json (hunk 2 above)
Comment (neutiyoo) [reply app/_locales/ko/messages.json:983]: I realized that solution B and C do not emphasize the negation so I came up with a A solution.
Unlike B or C, A emphasizes the negation with `절대` and is more close to the original one.

Therefore, the solution A is the more idiomatic one IMO.

Comment (neutiyoo) [reply app/_locales/ko/messages.json:983]: I realized that the previous solutions I mentioned(solution B and C) do not emphasize the negation.

Hence, I came up with the solution **A**.
Unlike B or C, A emphasizes the negation with `절대` making the meaning more close to the original one.

Therefore, the solution A is the correct and idiomatic one IMO.

Comment (Gudahtt) [reply app/_locales/ko/messages.json]: Great, thanks!

I have applied solution A in 0cb1ae01a530bd1508540ed02a0a2c96d115e7ea - let me know if I've made a mistake anywhere.

File: app/_locales/ko/messages.json [hunk 3]
```diff
@@ -989,118 +1489,433 @@
...
+    "message": "당사는 매번 최상의 유동성 소스에서 최적 가격을 찾습니다. $1% 요금은 각 견적에 자동으로 반영되어, MetaMask를 더욱 개선하는 현재 진행 중인 개발을 지원합니다.",
+    "description": "Provides information about the fee that metamask takes for swaps. $1 is a decimal number."
+  },
+  "swapNetworkFeeSummary": {
+    "message": "네트워크 요금에는 스왑을 처리하고 이더리움 네트워크에 보관하는 비용이 적용됩니다. MetaMask는 이 요금을 이용해 이득을 얻지 않습니다."
+  },
+  "swapNewQuoteIn": {
+    "message": "$1의 새 견적",
+    "description": "Tells the user the amount of time until the currently displayed quotes are update. $1 is a time that is counting down from 1:00 to 0:00"
+  },
```
[the remark this answers is not in the document]
Comment (Gudahtt) [reply app/_locales/ko/messages.json]: Thanks! I've applied the first solution you suggested here: 71a9f57c696db63891bc7f710696f284f6ee9afa

Please let me know if I've made a mistake, or if you think solution 2 would be more appropriate.

**Review by ghost — DISMISSED** [review]
LGTM

**Comment by Gudahtt** [comment]
This PR has been rebased onto `develop` before merging to ensure that the `yourSigRequested` message was not present (it was removed in #10300).

Thanks again for the review!

**Pull request merged**